App/Sagas/TextileSagas.js

```diff
@@ -155,14 +166,6 @@ export function * shareImage ({thread, hash}) {
   }
 }
 
-export function * refreshThreads ({newState}) {
```

Comment:
removed this and included in `photosTask` since it was causing a timing issue, even if the hookup in `index.js` was places after the one for `photosTask`, I think because the latter happens in a background task? Anyway, this just needs to happen after the node starts. Previously, this wasn't an issue cause the active state doesn't seem to be triggered on the first load... and for subsequent occurrences, although it was not working, the node was already "started".